src/player/player.service.js

```diff
@@ -0,0 +1,34 @@
+const axios = require("axios");
+const fs = require("fs");
+const path = require("path");
+
+class PlayerService {
+  constructor(apiUrl) {
+    // Example file
+    this.playerDataSetPath = path.resolve("./player-dataset.json");
+    this.awardPublisherUrl = apiUrl;
+  }
+
+  async getInfoSyncData() {
+    const playerInfoDataset = JSON.parse(
+      fs.readFileSync(this.playerDataSetPath, { encoding: "utf-8" })
+    );
+    return playerInfoDataset;
+  }
+
+  async updateBalance(signature, requestBody) {
+    const response = await axios.post(this.awardPublisherUrl, requestBody, {
+      headers: {
+        signature: signature,
```

Comment:
you already verify the signature in auth middleware, why do you need to send it again?

Reply:
I didn't change the logic we had in this file, just moved it in the better place, but I can fix this